### DISCUSSION TITLE
Adding multiple people to walletconnect project

### DISCUSSION THREAD

**Discussion by ghost** [WalletConnect/walletconnect-monorepo#3671] [Developer Support] [answered]
Hi, I’m having trouble adding other people to our wallet connect project. It is possible to add multiple people to a project? 

Or do you require people to all use the same account?

**Comment by glitch-txs** [comment, marked as answer]
Hi, it's not possible at the moment. But this feature is on our roadmap, no ETA atm. 🙏
